Libs/MRML/DisplayableManager/vtkThreeDViewInteractorStyle.cxx

```diff
@@ -792,6 +783,56 @@ void vtkThreeDViewInteractorStyle::SetModelDisplayableManager(
 }
 
 //----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse)
```

Comment:
Axis labels have been made configurable. In several places in the code, labels or orientations are still hardcoded, but we should really move away from that.

Therefore, we shouldn't use vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse), but something like this:

vtkThreeDViewInteractorStyle::RotateToViewAxis(const std::string& axisLabel)

axisLabel is "L", "R", ... is one of the labels in vtkMRMLAbstractViewNode::AxisLabels